.github/workflows/windows-test.yml

```diff
@@ -0,0 +1,47 @@
+name: Windows Pytest
+
+on:
+  push:
+    branches: [ master, release ]
+  pull_request:
+    branches: [ master, release ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: |
+          Data
+        key: notebook-data-{{ checksum "Data/manifest.json" }}
+        restore-keys: |
+          notebook-data-{{ checksum "Data/manifest.json" }}
+          notebook-data-
+    - name: Debug path
+      run: ls
+    - name: Debug data path
+      run: ls Data
+    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Remove the "Debug path" and "Debug data path" steps if not needed in the final script.